config/config.default.php

```diff
@@ -193,5 +193,11 @@
         'Contribution' => 'Please visit [engelsystem/engelsystem](https://github.com/engelsystem/engelsystem) if '
             . 'you want to to contribute, have found any [bugs](https://github.com/engelsystem/engelsystem/issues) '
             . 'or need help.'
+    ],
+    // Var dump server
+    'vardump_server' => [
+        'host' => '127.0.0.1',
+        'port' => '9912',
+        'enable' => true,
```

Comment:
This enables the functionality by default what should slow down development, i would just disable it by default

Reply:
My thought was to always have this prepared during development.
As for the performance side, this does not really have any significant impact.
I can however invert it

Reply:
That would be nice as i personally don't like it if software connects randomly to some ports before doing some fallback magic